packages/@aws-cdk/aws-glue/lib/connection.ts

```diff
@@ -0,0 +1,130 @@
+import { Construct, IResource, Resource, Aws } from '@aws-cdk/core';
+import ec2 = require('@aws-cdk/aws-ec2');
+import { CfnConnection } from './glue.generated';
+
+export enum ConnectionInputTypes {
+  /**
+   * The type of the connection. Currently, only JDBC is supported; SFTP is not supported.
+   */
+  JDBC = 'JDBC',
+}
+
+
+export interface IPhysicalRequirements {
+  /**
+   * @attribute
+   */
+  readonly availabilityZone?: string;
+
+  /**
+   * @attribute
+   */
+  readonly securityGroupIds?: ec2.ISecurityGroup[];
+
+  /**
+   * @attribute
+   */
+  readonly subnet?: ec2.ISubnet;
+}
+
+export interface IConnectionInput {
+  /**
+   * @attribute
+   */
+  readonly properties: object;
+
+  /**
+   * @attribute
+   */
+  readonly type: ConnectionInputTypes;
+
+  /**
+   * @attribute
+   */
+  readonly description?: string;
+
+  /**
+   * @attribute
+   */
+  readonly matchCriteria?: string[];
+
+  /**
+   * @attribute
+   */
+  readonly name?: string;
+
+  /**
+   * @attribute
+   */
+  readonly physicalRequirements?: IPhysicalRequirements;
+}
+
+export interface IConnection extends IResource {
+  /**
+   * @attribute
+   */
+  readonly catalogId: string;
+
+  /**
+   * @attribute
+   */
+  readonly connectionInput: IConnectionInput;
+
+  /**
+   * @attribute
+   */
+  readonly connectionName: string;
+}
+
+
+export interface ConnectionAttributes {
+  readonly catalogId: string;
+  readonly connectionInput: IConnectionInput;
+  readonly connectionName: string;
+}
+
+
+export interface ConnectionProps {
+  readonly catalogId?: string;
```

Comment:
Is it actually possible to provide this? Because in the docs below, it says this always has to be account ID.